app/models/booth.rb

```diff
@@ -1,5 +1,6 @@
 class Booth < ActiveRecord::Base
   include ActiveRecord::Transitions
+  has_paper_trail on: [:create, :update], ignore: [:updated_at], meta: { conference_id: :conference_id }
```

Comment:
I think `has_paper_trail ignore: [:updated_at], meta: { conference_id: :conference_id }` is enough